pkg/agent/util/ipset/ipset.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipset
+
+import (
+	"fmt"
+	"os/exec"
+	"regexp"
+	"strings"
+)
+
+type SetType string
+
+const (
+	// The hash:net set type uses a hash to store different sized IP network addresses.
+	// The lookup time grows linearly with the number of the different prefix values added to the set.
+	HashNet SetType = "hash:net"
+)
+
+// memberPattern is used to match the members part of ipset list result.
+var memberPattern = regexp.MustCompile("(?m)^(.*\n)*Members:\n")
+
+func CreateIPSet(name string, setType SetType) error {
+	args := []string{"create", name, string(setType), "-exist"}
+	cmd := exec.Command("ipset", args...)
```

Comment:
I'm probably missing something, but why not write this directly:
```
cmd := exec.Command("ipset", "create", name, string(setType), "-exist")
```
?

Reply:
Was copying some example code's style. Yes, one line is also very readable, merged.